### PR TITLE
Fixes to support multipod vm migration

### PR DESCRIPTION
**Pull request by akhilamohanan**
* added delay between dhcp release and renew
* multipod specific code will be executed only when aci-multipod is set

### CODE REVIEW

File: pkg/apicapi/apic_sync.go [hunk 1]
```diff
@@ -308,6 +308,10 @@ func (conn *ApicConnection) fullSync() {
 		"updates": len(updates),
 		"deletes": len(deletes),
 	}).Info("APIC full sync completed")
+
+	conn.SyncMutex.Lock()
+	conn.SyncDone = true
+	conn.SyncMutex.Unlock()
```
Comment (snaiksat) [inline pkg/apicapi/apic_sync.go:314]: I am missing the point of this lock.

**Review by snaiksat — APPROVED** [review]

File: pkg/apicapi/apic_sync.go (hunk 1 above)
Comment (akhilamohanan) [reply pkg/apicapi/apic_sync.go:314]: https://github.com/noironetworks/aci-containers/pull/1104/files#diff-d5c31f7209190065692316305836fc8a42834003000db2714b32ac867cc5c96fR373 - SyncDone is used here also and this is a different thread.

Comment (snaiksat) [reply pkg/apicapi/apic_sync.go:314]: What I meant to ask is that this seems to be an atomic action, so why is a lock required?

Comment (akhilamohanan) [reply pkg/apicapi/apic_sync.go:314]: Even if it is atomic, if we are writing to SyncDone without accessing lock, there is a chance of race if this shared variable is read in the other thread concurrently

**Pull request merged**